context_test.go

```
@@ -1,32 +1,47 @@
 
-	httpCtx := Background()
+	ginCtx, _ := gin.CreateTestContext(httptest.NewRecorder())
+	ginCtx.Request = httptest.NewRequest("GET", "/", nil)
+	httpCtx := NewContext(ginCtx)
 	httpCtx.WithValue("Hello", "world")
 	httpCtx.WithValue("Hi", "Goravel")
 	httpCtx.WithValue(customTypeKey, "halo")
-	httpCtx.WithValue(anotherCustomTypeKey, "hola")
+
+	userContext := context.Background()
+	userContext = context.WithValue(userContext, "user_a", "b")
```

golangci-lint (1.62.2)

SA1029: should not use built-in type string as key for value; define your own type to avoid collisions

(staticcheck)